src/components/AppBar/AppBar.component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { CssBaseline, AppBar, Toolbar, Typography, Link } from '@material-ui/core';
+import YouTubeIcon from '@material-ui/icons/YouTube';
+import SearchBar from '../SearchBar';
+import User from '../User';
+import './AppBar.styles.css';
+
+export default function App() {
+  return (
+    <>
+      <CssBaseline />
+      <AppBar position="fixed" color="default">
+        <Toolbar>
+          <Link edge="start" href="/" target="_self" rel="noopener noreferrer">
+            <YouTubeIcon color="secondary" fontSize="large" />
+          </Link>
```

Comment:
Instead of using `href` property, you should use `to` because every time the user clicks on the YoutubeIcon a refresh is being taken on the browser (bad user experience). Using to as the property, won't make a refresh of the page and will tell react-router-dom to change the UI.